app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: None) -> None:
+    func_results = {}
+
+    def wrapper(*args, **kwargs) -> None:
+        key = (args, frozenset(kwargs.items()))
+
+        if key in func_results:
+            print("Getting from cache")
+            return func_results[key]
+        else:
```

Comment:
There is no need for `else`, the code will not execute after the `return` statement.

Reply:
done